easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-6.0-centos6_x86_64.eb

```diff
@@ -0,0 +1,44 @@
+name = 'FreeSurfer'
+version = '6.0.0'
+versionsuffix = '-centos6_x86_64'
+
+homepage = 'http://surfer.nmr.mgh.harvard.edu/fswiki/FreeSurferWiki'
+description = """FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data.
+FreeSurfer contains a fully automatic structural imaging stream for processing cross sectional and longitudinal data."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(namelower)s-Linux%(versionsuffix)s-stable-pub-v%(version)s.tar.gz',
+           'runtime2012bLinux.tar.gz']
+source_urls = ['ftp://surfer.nmr.mgh.harvard.edu/pub/dist/%(namelower)s/%(version)s',
+               'http://surfer.nmr.mgh.harvard.edu/fswiki/MatlabRuntime?action=AttachFile&do=get&target=']
+
+# see https://surfer.nmr.mgh.harvard.edu/fswiki/SetupConfiguration_Linux
+modextravars = { 
+    'FREESURFER_HOME': '$root',
+    'FSFAST_HOME': '$root/fsfast',
+    'FSF_OUTPUT_FORMAT': 'nii.gz',
+    'SUBJECTS_DIR': '$root/subjects',
+    'MNI_DIR': '$root/mni',
+    'FS_OVERRIDE': '0',
+    'PERL5LIB': '$root/mni/lib/perl5/5.8.5',
+    'OS': 'linux',
+    'MNI_PERL5LIB': '$root/mni/lib/perl5/5.8.5',
+    'FMRI_ANALYSIS_DIR': '$root/fsfast',
+    'MINC_BIN_DIR': '$root/mni/bin',
+    'MINC_LIB_DIR': '$root/mni/lib',
+    'MNI_DIR': '$root/mni',
+    'MNI_DATAPATH': '$root/mni/data',
+}
+
+modextrapaths = {
+     'PATH': 'bin:$root/tktools:$root/fsfast/bin:$root/mni/bin',
+}
+
+postinstallcmds = ['cp -rf %(builddir)s/MCRv80/ %(installdir)s']
```

Comment:
Also, what does this provide exactly, just the MATLAB Runtime? If so, we should list `MCR` as a dependency instead?